src/utility/watchdog/Watchdog.cpp

```diff
@@ -63,6 +66,11 @@ static void samd_watchdog_reset()
   }
 }
 
+static void samd_watchdog_enable_network_feed()
+{
+  WiFi.setFeedWatchdogFunc(watchdog_reset);
```

Comment:
This function works for NINA? Not to say that it does not, I just never saw it before :wink:

Reply:
Yes, we added this function with this PR https://github.com/arduino-libraries/WiFiNINA/pull/186 one year ago